.github/workflows/tests.yaml

```diff
@@ -32,8 +32,20 @@ jobs:
       - name: Setup
         uses: ./.github/actions/setup
 
-      - name: Install dependencies
```

Comment:
Removed this step because dep installation happens in `Setup` as well so this was just burning compute